web/packages/teleterm/src/deepLinks.test.ts

```diff
@@ -136,20 +139,32 @@ describe('parseDeepLink', () => {
 
 describe('makeDeepLinkWithSafeInput followed by parseDeepLink gives the same result', () => {
   const inputs: Array<Parameters<typeof makeDeepLinkWithSafeInput>[0]> = [
-    {
-      proxyHost: 'cluster.example.com',
-      path: Path.ConnectMyComputer,
-      username: undefined,
-    },
-    {
-      proxyHost: 'cluster.example.com',
-      path: Path.ConnectMyComputer,
-      username: 'alice',
-    },
+    // {
+    //   proxyHost: 'cluster.example.com',
+    //   path: '/connect_my_computer',
+    //   username: undefined,
+    //   searchParams: {},
+    // },
+    // {
+    //   proxyHost: 'cluster.example.com',
+    //   path: '/connect_my_computer',
+    //   username: 'alice',
+    //   searchParams: {},
+    // },
+    // {
+    //   proxyHost: 'cluster.example.com:1337',
+    //   path: '/connect_my_computer',
+    //   username: 'alice.bobson@example.com',
+    //   searchParams: {},
+    // },
```

Comment:
Is something wrong with these tests?

Reply:
Yeah... they are commented out 😅 , will uncomment. thank you!